src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -114,7 +118,12 @@ public void run() {
                         LOGGER.log(Level.WARNING, "Failed to parse the polling log",e);
                         cause = new GitHubPushCause(pushBy);
                     }
-                    if (job.scheduleBuild(cause)) {
+                    ParameterizedJobMixIn scheduler = new ParameterizedJobMixIn() {
+                        @Override protected Job asJob() {
```

Comment:
please split `@Override` annotation and method signature to two lines as more preferable formatting way